crates/router/src/services/api.rs

```diff
@@ -133,8 +137,8 @@ pub trait ConnectorIntegration<T, Req, Resp>: ConnectorIntegrationAny<T, Req, Re
         &self,
         _req: &types::RouterData<T, Req, Resp>,
         _connectors: &Connectors,
-    ) -> CustomResult<Option<types::RequestBody>, errors::ConnectorError> {
-        Ok(None)
+    ) -> CustomResult<RequestContent, errors::ConnectorError> {
+        Ok(RequestContent::Json(Box::new(json!(r#""#))))
```

Comment:
Since this method will return http request body, let's default it to empty json object instead of empty string
```suggestion
Ok(RequestContent::Json(Box::new(json!(r#"{}"#))))
```